### ISSUE TITLE
Add performance metrics

### ISSUE THREAD

**Issue by param087** [param087/swiftML#43]
**Metrics**

* Classification
  - [ ] accuracyScore
  - [ ] balancedAccuracyScore
  - [ ] averagePrecisionScore
  - [ ] brierScoreLoss
  - [ ] f1Score
  - [ ] logScore
  - [ ] precisionScore
  - [ ] recallScore
  - [ ] jaccardScore
  - [ ] rocAucScore

* Regression
  - [ ] explainedVarianceScore
  - [ ] maxError
  - [ ] meanAbsoluteError
  - [ ] meanSquaredError
  - [ ] meanSquaredLogError
  - [ ] medianAbsoluteError
  - [ ] r2Score

* Clustering
  - [ ] adjustedMutualInfoScore
  - [ ] adjustedRandScore
  - [ ] completenessScore
  - [ ] fowlkesMallowsScore
  - [ ] homogeneityScore
  - [ ] mutualInfoScore
  - [ ] normalizedMutualInfoScore
  - [ ] vMeasureScore

Reference implementation - https://github.com/scikit-learn/scikit-learn/tree/master/sklearn/metrics

**Comment by olegtaratuhin** [comment]
Hello! I would like to help with this issue

**Comment by param087** [comment]
@olegtaratuhin go ahead. If you need any help, please reach out.

**Comment by das-ankur** [comment]
Hello, I want to work on this issue.

**Comment by param087** [comment]
@das-ankur, you can look into scikit-learn for reference.

**Comment by das-ankur** [comment]
@param087 Thank you.

**Comment by das-ankur** [comment]
@param087 Hello, I need some help to install Tensorflow for swift on windows 10.  Tensorflow is already in my pc and I use it using python but I need some help to install it for swift.

**Comment by param087** [comment]
@das-ankur, you can find the details to install Swift for TensorFlow in the following link:
https://github.com/tensorflow/swift/blob/main/Installation.md
